lastuserapp/__init__.py

```diff
@@ -32,7 +32,7 @@
 db.init_app(app)
 db.app = app  # To make it work without an app context
 migrate = Migrate(app, db)
-RQ(app)  # Pick up RQ configuration from the app
+rq = RQ(app)  # Pick up RQ configuration from the app
```

Comment:
Does RQ support `init_app`? If so, move the object to the `lastuser_oauth` package, import it here, and do `init_app(app)`.